src/lib/webworker.generated.d.ts

```diff
@@ -430,12 +430,10 @@ interface ImageData {
     data: number[];
     height: number;
     width: number;
+    new(width: number, height: number): ImageData;

```

Comment:
this is going to be a breaking change. are we sure that ImageData does not have a constructor overload that takes no arguments?